docs/en/sql-reference/sql-statements/Data Manipulation/ROUTINE LOAD.md

```diff
@@ -163,10 +163,21 @@ FROM data_source
 
         Whether to enable strict mode, the default is on. If turned on, the column type transformation of non-null raw data is filtered if the result is NULL. Specified as "strict_mode" = "true"
             
-    5. timezone
+    5. `timezone`
 
         Specifies the time zone in which the job will be loaded. The default by using session variable's timezone. This parameter affects all function results related to the time zone involved in the load.
 
+    6. `format`
+
+        Specifies the format of the imported data. Support csv and json, the default is csv.
+
+    7. `jsonpaths`
+
+        Set jsonpath for precise imports, otherwise simple imports.
```

Comment:
what does `simple imports` mean? Should specify that `jsonpaths` is only valid when format is `json`

Reply:
i had add more comment